croptalk/evaluate_overall_performance.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+os.chdir('/app')
+
+from langchain.smith import RunEvalConfig
+from langsmith import Client
+
+from croptalk.model_openai_functions import model as MODEL
+
+from dotenv import load_dotenv
+
+load_dotenv('secrets/.env.secret')
+load_dotenv('secrets/.env.shared')
+
+DATASET_NAME = "sp-cp-bp-cih-70"
+
+# DOCS: 
+# https://api.python.langchain.com/en/latest/smith/langchain.smith.evaluation.config.RunEvalConfig.html
+
+eval_config = RunEvalConfig(
```

Comment:
An LLM will "decide" if ref answer and current answer are "close enough" to determine correctness, is that it? Do we know how exactly?

Reply:
We have to read on that. For now, I'm using default QA evaluator from langsmith. We definitely can improve it and create custom evaluators later on